ledger-core-ripple/test/Common.cpp

```diff
@@ -0,0 +1,6 @@
+#include "Common.hpp"
+
+namespace ledger {
+    namespace core {
+    }
+}
```

Comment:
What is it for ?

Reply:
The .hpp is used by several integration tests, but good call, that .cpp can be removed.